test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,65 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import java.util.*;
```

Comment:
Imports from java and third parties must come before `seedu`, and separated with line breaks :)